resolwe_bio/processes/support_processors/archive.yml

```diff
@@ -63,13 +63,23 @@
         {% endfor %}
 
         # Select data for IGV and organize by species / build.
+        IGV_FILE={{ d.species }}_{{ d.build }}_temp_igv.txt
+
         {% if (d|type).startswith('data:alignment:bam') %}
           mv {{ d|sample_slug }}/bai/{{ d.bai.file|basename }} {{d|sample_slug}}/bam/{{ d.bai.file|basename }}
           rm -r {{d|sample_slug}}/bai
-          echo {{ d|sample_slug }}/bam/{{ d.bam.file|basename }} >> {{ d.species }}_{{ d.build }}_temp_igv.txt
+          # In the first line of _temp_igv.txt file add build information and than paths to files.
```

Comment:
I would rephrase this to "Add build information in the first line of _temp_igv.txt file. In the following lines add paths to bam files."